lib/features/pokemon_overview/widget/pokemon_grid_view.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:pokedex_start/api/model/pokemon.dart';
+import 'package:pokedex_start/features/pokemon_overview/widget/pokemon_card.dart';
+import 'package:pokedex_start/utilities/constant.dart';
+
+import 'package:flutter/material.dart';
+
+class PokemonGridView extends StatelessWidget {
+  const PokemonGridView({
+    super.key,
+    required this.pokemons,
+  });
+
+  final List<Pokemon> pokemons;
+
+  @override
+  Widget build(BuildContext context) {
+    return GridView.builder(
+      itemCount: pokemons.length,
+      gridDelegate: const SliverGridDelegateWithFixedCrossAxisCount(
+        crossAxisCount: gridRowCount,
+        childAspectRatio: cardRatio,
+      ),
+      itemBuilder: (BuildContext context, int index) {
```

Comment:
no need to add its object type 🙂 
also, kindly rename `context` to `_` since it's unused 🙂